docs/source/contents/4_draft_mapping.rst

```diff
@@ -2152,6 +2152,622 @@ Drop all values in mods:part.
+        relators:cre <http://id.loc.gov/authorities/names/n83189337> ;
+        relators:cre <http://id.loc.gov/authorities/names/n84053297> ;
+        relators:cre <http://id.loc.gov/authorities/names/no2004018542> ;
+        relators:cre <http://id.loc.gov/authorities/names/n83189337> ;
+        relators:cre <http://id.loc.gov/authorities/names/no90027536> ;
+        relators:cre <http://id.loc.gov/authorities/names/n82020447> ;
+        relators:cre <http://id.loc.gov/authorities/names/n87856030> ;
+        relators:cre <http://id.loc.gov/authorities/names/n85031246> ;
+        relators:cre <http://id.loc.gov/authorities/names/n84053297> ;
+        relators:cre <http://id.loc.gov/authorities/names/no2001006452> ;
+        relators:cre <http://id.loc.gov/authorities/names/n81119648> ;
+        relators:cre "Jensen, Richard E." ;
+        relators:cre <http://id.loc.gov/authorities/names/no2004118058> ;
+        relators:cre <http://id.loc.gov/authorities/names/no2004118055> ;
+        relators:cre "Schock, Jack" ;
+        relators:cre <http://id.loc.gov/authorities/names/n88243079> ;
+        relators:cre <http://id.loc.gov/authorities/names/n78078895> ;
+        relators:cre <http://id.loc.gov/authorities/names/n50038965> .
+
 Hierarchical Sheet Music Identifier
```

Comment:
I had added this to relatedItem when we were looking over identifiers as we had identified the sheetmusic_hostItem property from opaquenamespace identifier then. You have already accounted for this in the relatedItem/identifier[@type] category. Feel free to remove this from the mapping.

Reply:
I feel like a real doofus: I completely failed to read past my own work here. Apologies.